packages/editor/src/components/error-boundary/style.scss

```diff
@@ -2,6 +2,6 @@
 	margin: auto;
 	max-width: 780px;
 	padding: 20px;
-	margin-top: 60px;
+	margin-top: $header-height;
```

Comment:
Same as [this comment](https://github.com/WordPress/gutenberg/pull/60729/files#r1564473696)